src/sql/workbench/parts/dataExplorer/common/nodeContext.ts

```diff
@@ -86,4 +107,36 @@ export class NodeContextKey extends Disposable implements IContextKey<INodeConte
 	get(): INodeContextValue | undefined {
 		return this._nodeContextKey.get();
 	}
+
+	/**
+	 * Helper function to get the correct context menu for showing
+	 * extension actions
+	 */
+	public async setExtensionContextKeys(): Promise<void> {
+		for (let extension of inBuiltExtensions) {
+			let extensionLoaded = await this.extensionService.getExtension(extension);
```

Comment:
What about if a user installs/uninstalls an extension? We don't reload ADS for the majority of them anymore so the menu items I wouldn't expect to show up in that case which isn't very desireable.